README.md

```diff
@@ -90,6 +91,10 @@ A curated list of awesome Java frameworks, libraries and software.
 - [Orika](https://github.com/orika-mapper/orika) - JavaBean-mapping framework that recursively copies (among other capabilities) data from one object to another.
 - [Selma](https://github.com/xebia-france/selma) - Annotation processor-based bean mapper.
 
+### Blockchain
+
+- [web3j](https://github.com/web3j/web3j) - Lightweight library for integrating with the Ethereum blockchain.
```

Comment:
Add more blockchain-related libraries or move it to the Misc section until we find enought libraries, see CONTRIBUTE.md